dist/rxp-js.js

```diff
@@ -427,28 +527,71 @@ var RealexHpp = (function () {
 			return tabWindow;
 		},
 
+		/**
+		 * Creates a rudimentary URL parser using an anchor element
+		 *
+		 * @param {string} url
+		 * @returns the created anchor element
+		 */
 		getUrlParser: function (url) {
 			var parser = document.createElement('a');
 			parser.href = url;
 			return parser;
 		},
 
+		/**
+		 * Gets the hostname/origin from a URL. Used for origin checks
+		 *
+		 * @param {string} url
+		 * @returns the hostname/origin of the URL
+		 */
 		getHostnameFromUrl: function (url) {
-      return internal.getUrlParser(url).hostname;
+			return internal.getUrlParser(url).hostname;
 		},
 
+		/**
+		 * Compares the origins from both arguments to validate we have received a postMessage
+		 * from the expected source
+		 *
+		 * @param {string} origin The origin attached to the recieved message
+		 * @param {string} hppUrl Our expected source origin
+		 * @returns true if the origins match
+		 */
 		isMessageFromHpp: function (origin, hppUrl) {
 			return internal.getHostnameFromUrl(origin) === internal.getHostnameFromUrl(hppUrl);
 		},
 
+		/**
+		 * Handles messages from the HPP
+		 *
+		 * Messages from the HPP are one of:
+		 *
+		 * - iframe resize event
+		 * - transaction response
+		 * - error information
+		 *
+		 * @param {MessageEvent} e
+		 */
 		receiveMessage: function (e) {
 			//Check the origin of the response comes from HPP
 			if (!internal.isMessageFromHpp(e.event.origin, hppUrl)) {
 				return;
 			}
+
+			if (!e.event.data) {
+				return;
+			}
+
+			var evtdata = internal.decodeAnswer(e.event.data);
+
+			// we received an invalid message from the HPP iframe (e.g. from a browser plugin)
```

Comment:
If anyone is coming here with the hopes of using the new version of 1.5.0 upgrading from 1.3.1 and getting it that any keyboard key is causing a submit, this is the solution.
It does appear to be a browser extension that is causing this issue and this code fixes this problem.
Thanks for submitting it has saved me after hours of thinking I had done something wrong.
@RealexITSO this needs to be implimented as of now v1.5.0 is broken!